torch/csrc/jit/codegen/cuda/scheduler/utils.h

```diff
@@ -248,6 +248,23 @@ struct BroadcastMultiple {
 // data type size.
 std::vector<BroadcastMultiple> getBroadcastMultiples(TensorView* reference_tv);
 
+namespace gemm_utils {
```

Comment:
Are you using `mma` and `gemm` interchangeably? If so, can we stick to one of them?

Reply:
I don't think they are interchangeable.

Gemm is a generic family of subroutines that exists in a much wider context. Mma is the tensorcore intrinsic and is discussed only within the GPU context. We could implement gemm with or without mma and similarly we could implement kernels that aren't gemm with mma. 

Most of the logic in this PR is meant to enable mma usage in the generated kernels. And in this file `gemm_utils` is just a few scheduler routines that facilitates tiling a gemm or a gemm-like kernel, and is relatively less mma-related. 

I guess we would see more and more decoupling between the two as we build out.

Reply:
I see another concern is mma builder taking gemm_tile options as an input. That isn't necessary and I will separate them out. It'd just be much easier to do after the swizzle op PR.

Reply:
I was asking if you're using the two words interchangeably because technically speaking both of them didn't appeared to be used consistently. If mma is a GPU-specific intrinsic, the `mma` function in arith.h should be renamed to something else. "mma" should be used only in the context of scheduling.

Also, strictly speaking, GEMM does accumulation, so I assume that's not expressible within our fusion IR. It doesn't sound right to me to say `gemm_utils` even though we don't strictly support GEMM.

Reply:
Sure I could rename `gemm_utils` to `matmul_tile_utils` if that sounds better. Supporting accumulation is a separate topic I guess. 

The arith interface can be `matmul` and I guess we can have the non-mma lowering path in a separate PR. Not sure if there's a better name for MmaOp fusion IR though.

Reply:
updated naming. resolving.